WordPress/Classes/ViewRelated/Stats/Operation/AsyncBlockOperation.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+class AsyncBlockOperation: AsyncOperation {
```

Comment:
`AsyncBlockOperation` is an opposite to [`BlockOperation`](https://developer.apple.com/documentation/foundation/blockoperation) which is a synchronous operation. It kinds of carry over the term "block" from the Foundation type.

Reply:
> `AsyncBlockOperation` adds convenience of not having to create a dedicated subclass...

Yes, that's the idea. It's nice in some occasions to not having to create a type to run some arbitrary code. But I don't think it's suitable for subclass though—it would be easier to just extend from `AsyncOperation`. Same way that you would extend from `Operation`, rather than `BlockOperation`.

Reply:
I didn't know or remember about [`BlockOperation`](https://developer.apple.com/documentation/foundation/blockoperation). That "prior art" adds more context to this name.

Also, even though Swift is getting close to its tenth anniversary, I doubt there are many developers that are not familiar  with the meaning of "block" in the Objective-C context, so, even if the name is not immediately straightforward, it's not a stretch to understand what it might be about. I was being pedantic, as usual.

> But I don't think it [`AsyncBlockOperation`] is suitable for subclass though

Neither do it. If one needs more advanced customization, they should subclass `Operation` or `AsyncOperation`.